portability-gateway/src/main/java/org/dataportabilityproject/gateway/ApiFlags.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dataportabilityproject.gateway;
```

Comment:
since (I think) we are going to still call this binary the "api", should the package be named api instead of gateway? We can still keep the top-level portability-gateway directory during the migration, and rename it api after the old code is gone.